pkg/couchdb/couchdb.go

```diff
@@ -839,6 +857,30 @@ func DefineIndexes(g *errgroup.Group, db Database, indexes []*mango.Index) {
 	}
 }
 
+// Copy copies an existing doc to a specified destination
+func Copy(db Database, doctype, path, destination string) (map[string]interface{}, error) {
+	headers := make(map[string]string)
+	headers["Destination"] = destination
+	headers["Content-Type"] = "application/json"
```

Comment:
Why do you add  a `Content-Type: application/json` here? I don't see a body in the request

Reply:
Right, it's a test residual, my bad